apps-rendering/src/components/Headline/index.tsx

```diff
@@ -42,8 +43,20 @@ const Headline: React.FC<Props> = ({ item }) => {
 		case ArticleDesign.LiveBlog:
 		case ArticleDesign.DeadBlog:
 			return <BlogHeadline item={item} />;
-		case ArticleDesign.Interview:
-			return <InterviewHeadline item={item} />;
+		case ArticleDesign.Interview: {
+			const designTag = <DesignTag format={format} />;
+			return (
+				<>
+					<nav>
+						<a href={'https://www.theguardian.com/tone/interview'}>
+							{designTag}
+						</a>
+					</nav>
```

Comment:
If the only place `DesignTag` is used in this file is for `InterviewHeadline` I wonder if this should live in the `InterviewHeadline` component? So far the convention for the `index`/`defaults` pattern has been to select sub-components using the `index` file, and let those sub-components manage layout and styles. However, whether that pattern is intuitive and works in practice is up for debate. Do you have any thoughts?

Reply:
Thanks Jamie for pairing and helping understand what the `index / defaults` pattern and it's purpose. I moved the design tag to the sub component `InterviewHeadline` 👍